shared/js/blobview.js

```diff
@@ -260,114 +260,26 @@ var BlobView = (function() {
 
     getASCIIText: function(offset, len) {
       var bytes = new Uint8Array(this.buffer, offset + this.viewOffset, len);
-      return String.fromCharCode.apply(String, bytes);
+      var decoder = new TextDecoder('ascii');
+      return decoder.decode(bytes);
```

Comment:
As you found in image_size.js, this new implementation is not equivalent to the old. Arguably, the original method should have been getBinaryString().  Feel free to rename it if you want to. But I'm reluctant to just change this method. Is there any reason that TextDecoder('ascii') is actually needed here? I'd guess that String.fromCharCode() is actually the faster option here, and would recommend that you just don't change this at all.